src/styles/blocks/prod-card.css

```diff
@@ -0,0 +1,100 @@
+.prod-card {
+  --color-dark: #060b35;
+  --color-light: #616070;
```

Comment:
declare colors in page.css?

Reply:
yep, it's better to write variables in main page or in separate file for variables

Reply:
> yep, it's better to write variables in main page or in separate file for variables

Yeah, but if I want to use this block in other project, how my block get access to this variables?

Reply:
Then you can import variables files there too 🙂